README.md

```diff
@@ -33,7 +33,21 @@ To enable testing in the app, your Rails app needs:
 the dimension to use in Google Analytics
 3. A response HTTP header that tells Fastly you're doing an A/B test
 
-Let's say you have this controller:
+Start by defining which acceptance testing framework you will use. This gem
+supports both Capybara and ActiveSupport. In order to configure it, add this to
+your test helper file:
+
+```
+GovukAbTesting.configure do |config|
+  config.acceptance_test_framework = :capybara # or :active_support
```

Comment:
I don't think we are in a position to dictate a default because of it varies across apps. The gem will raise an exception if it's not set, which I think is a sensible way of reminding us to set one.